### ISSUE TITLE
Image Tour - Place lost in tour

### ISSUE THREAD

**Issue by atiro** [digirati-co-uk/iiif-manifest-editor-components#35]
Inconsistent one and hard to reproduce but sometimes going back from Free explore to image tour has lost it's place and so the tour seems to have ended (but hasn't). Possibly a Mac bug (seen twice on mac's, not an issue in Firefox/Linux)

**Comment by stephenwf** [comment]
@atiro Do you have a link to an example of this, or the JSON of a manifest that I can use to replicate this?

**Comment by atiro** [comment]
I don't think we can remember now. Happy to close (the only live tour is https://www.vam.ac.uk/articles/pietre-dure-the-tomb-of-cecilia-metella)

**Comment by stephenwf** [comment]
@atiro thanks, I was able to replicate on the link you sent above.

1) Scroll down to the very bottom of the page
2) Click "free explore"
3) Move the camera around

**Expected:** the position should revert back to the last position
**Actual:** the position does not change

However, as soon as you scroll it does jump back correctly and only happens on the last one.
